src/sidebar/components/test/annotation-timestamps-test.js

```diff
@@ -87,9 +87,10 @@ describe('AnnotationTimestamps', () => {
   });
 
   it('is updated after time passes', () => {
-    fakeTime.decayingInterval.callsFake((date, callback) =>
-      setTimeout(callback, 10)
```

Comment:
This mock for `decayingInterval` returned the result of `setTimeout`, which is a timer ID. The real function returns a cleanup function, which is compatible with `useEffect`, so I've modified the mock to do the same.